packages/env/lib/build-docker-compose-config.js

```diff
@@ -181,6 +181,16 @@ module.exports = function buildDockerComposeConfig( config ) {
 		config.env.tests.phpmyadminPort ?? ''
 	}}:80`;
 
+	const healthcheck = {
+		test: [
+			'CMD-SHELL',
+			'if [ "$LOCAL_DB_TYPE" = "mariadb" ]; then case "$LOCAL_DB_VERSION" in 5.5|10.0|10.1|10.2|10.3) mysqladmin ping -h localhost || exit $?;; *) mariadb-admin ping -h localhost || exit $?;; esac; fi',
```

Comment:
I'm not very familiar with Docker, but I think `LOCAL_DB_TYPE` and `LOCAL_DB_VERSION` are environment variables for core development. I think [the image is always `mariadb` in `wp-env`](https://github.com/WordPress/gutenberg/blob/36c682d6e9586ee3653e971b661591d8a6df7875/packages/env/lib/build-docker-compose-config.js#L197), so can we simplify the command a bit?